### PR TITLE
Please review it first before merging!!! easy_install was commented out or replaced with pip.

### PR DESCRIPTION
**Pull request by Muntasib-creator**

### CODE REVIEW

**Review by riz-hossain — CHANGES_REQUESTED** [review]
Remove any debug print statements

File: ZeuZ_Node_Installer.py [hunk 1]
```diff
@@ -163,7 +163,7 @@ def get_required_mods():
             else:
                 print("Error - Need root privleges\n")
                 quit()
-
+            print("Asche")
             install(type="sudo", cmd="easy_install pip")
```
Comment (riz-hossain) [inline ZeuZ_Node_Installer.py]: Remove debuging print

Comment (Muntasib-creator) [reply ZeuZ_Node_Installer.py]: Done removing

Comment (Muntasib-creator) [reply ZeuZ_Node_Installer.py]: Dont merge it. Its not been tested. Just solved all the issues on my laptop so just saved the codes in separate branch

**Review by riz-hossain — APPROVED** [review]

**Pull request merged**